vmdb/app/controllers/miq_policy_controller/rsop.rb

```diff
@@ -137,6 +132,13 @@ def rsop_show_options
 
   private
 
+  def rsop_put_objects_in_sb(objects, key)
+    @sb[:rsop][key] = {}
+    objects
+      .sort_by { |o| o.name.downcase }
```

Comment:
I've also seen this done another way. But I'm not sure which generates less garbage.

```ruby
@sb[:rsop][key] = Hash[objects.sort_by { |o| o.name.downcase }.collect{|o| [i.id.to_s, o.name] }]
```